### ISSUE TITLE
[examples] Support ResNet 50 v2 OpenVino model for image classification examples.

### ISSUE THREAD

**Issue by BruceDai** [intel/webml-polyfill#1072]
Since there isn't available ResNet 50 v2 OpenVino model, we try to following workaround to generate ResNet 50 v2 OpenVino model:
- Converted [resnet50v2.onnx](https://s3.amazonaws.com/onnx-model-zoo/resnet/resnet50v2/resnet50v2.onnx) of [ONNX Model Zoo](https://github.com/onnx/models/tree/master/vision/classification/resnet) to OpenVino format by following [Converting a ONNX* Model](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_ONNX.html) guide of OpenVINO Toolkit, the inferenced label wasn't correct. Converted model XML file [here](https://drive.google.com/file/d/1-WJqwEkX-mV3JGDSGe35adBs55fJTm74/view?usp=sharing).

- Converted [resnet_v2_50.ckpt](http://download.tensorflow.org/models/resnet_v2_50_2017_04_14.tar.gz) of [TensorFlow Models](https://github.com/tensorflow/models/tree/master/research/slim#pre-trained-models) to OpenVino format by following [Converting TensorFlow*-Slim Image Classification Model Library Models](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_convert_model_tf_specific_Convert_Slim_Library_Models.html) guide of OpenVINO Toolkit, the inferenced label also wasn't correct. Converted model XML file [here](https://drive.google.com/file/d/1XEZjRWCsycK5KctkA89uv0iKIkEQmBLt/view?usp=sharing).

BTW, also tried to convert  [resnet50v1.onnx](https://s3.amazonaws.com/onnx-model-zoo/resnet/resnet50v1/resnet50v1.onnx) to get ResNet 50 v1 OpenVino model, the inferenced label wasn't correct.

Synced with @pinzhenx, he last converted ResNet 50 v1 OpenVino model also by using above second workaround,  I will add the step of freezing graph refer to https://github.com/tensorflow/models/issues/4001

**Comment by BruceDai** [comment]
The inference result(label) wasn't expected even though adding the step of freezing graph. 
I'm ongoing to investigate this issue. 
Next: Convert ResNet 50 v1 OpenVino model by its checkpoint, and compare with our existed using esNet 50 v1 OpenVino model to find the root cause of unexpected result by converted ResNet 50 v2 OpenVino model.
